Scheduling.h

```diff
@@ -38,7 +39,7 @@ typedef struct {
    int priority;
 } SchedulingArg;
 
-bool Scheduling_setPolicy(Process* proc, Arg arg);
+bool Scheduling_setPolicy(Row* proc, Arg arg);
```

Comment:
Keep API to `Process*`. The caller needs to ensure we got a process.

Reply:
Reworked this now, thanks.